### ISSUE TITLE
Y axis label on facet_row subplots incorrect

### ISSUE THREAD

**Issue by ricecooker** [plotly/plotly_express#129]
The traces and the y-axis label on the subplots seem to be off by one when using facet_row to generate subplots. See image below:

<img width="739" alt="Screen Shot 2019-08-05 at 4 42 25 PM" src="https://user-images.githubusercontent.com/1724034/62493877-2aaad580-b7a0-11e9-8612-1da50045e39a.png">

The code to recreate this is below:

```python
import pandas.util.testing as put
import plotly.express as px
sample_df = put.makeTimeDataFrame().reset_index().melt(id_vars='index')
px.line(sample_df, x='index', y='value', facet_row='variable')
```

Installed via `pip install plotly==4.0`
```
>>> px.__version__
'0.3.0'
```

**Comment by nicolaskruchten** [comment]
Yikes, that's a pretty serious problem. We'll get it fixed shortly. Thanks for bringing it to our attention!

**Comment by jonmmease** [comment]
Fixed by https://github.com/plotly/plotly.py/pull/1704/commits/c75fe3e0594b20287b474db4455f86f4a9f92747 and will be released in 4.1.0 shortly.
